src/PhpWord/Element/CheckBox.php

```diff
@@ -71,4 +78,26 @@ public function getName()
     {
         return $this->name;
     }
+
+    /**
+     * Set default state.
+     *
+     * @return self
+     */
+    public function setDefaultChecked(bool $default = true)
```

Comment:
```suggestion
    public function setDefaultChecked(bool $default = true): self
```

Reply:
How can i update the changelog ?

Reply:
I added the test but i dont know why it fails only on 8.2